transfer.go

```diff
@@ -38,24 +38,25 @@ type TransferListParams struct {
 // Transfer is the resource representing a Stripe transfer.
 // For more details see https://stripe.com/docs/api#transfers.
 type Transfer struct {
-	ID        string            `json:"id"`
-	Live      bool              `json:"livemode"`
-	Amount    int64             `json:"amount"`
-	Currency  Currency          `json:"currency"`
-	Created   int64             `json:"created"`
-	Date      int64             `json:"date"`
-	Desc      string            `json:"description"`
-	FailCode  TransferFailCode  `json:"failure_code"`
-	FailMsg   string            `json:"failure_message"`
-	Status    TransferStatus    `json:"status"`
-	Type      TransferType      `json:"type"`
-	Tx        *Transaction      `json:"balance_transaction"`
-	Meta      map[string]string `json:"metadata"`
-	Bank      *BankAccount      `json:"bank_account"`
-	Card      *Card             `json:"card"`
-	Recipient *Recipient        `json:"recipient"`
-	Statement string            `json:"statement_descriptor"`
-	Reversals *ReversalList     `json:"reversals"`
+	ID                string            `json:"id"`
+	Live              bool              `json:"livemode"`
+	Amount            int64             `json:"amount"`
+	Currency          Currency          `json:"currency"`
+	Created           int64             `json:"created"`
+	Date              int64             `json:"date"`
+	Desc              string            `json:"description"`
+	FailCode          TransferFailCode  `json:"failure_code"`
+	FailMsg           string            `json:"failure_message"`
+	Status            TransferStatus    `json:"status"`
+	Type              TransferType      `json:"type"`
+	Tx                *Transaction      `json:"balance_transaction"`
+	Meta              map[string]string `json:"metadata"`
+	Bank              *BankAccount      `json:"bank_account"`
+	Card              *Card             `json:"card"`
+	Recipient         *Recipient        `json:"recipient"`
+	Statement         string            `json:"statement_descriptor"`
+	Reversals         *ReversalList     `json:"reversals"`
+	SourceTransaction *Transaction      `json:"source_transaction"`
```

Comment:
I'm not crazy about the existing naming, but what do you about renaming this to `SourceTx` for consistency with the `Tx` field?